app/views/partials/swagger/operations/operation.hbs

```diff
@@ -25,15 +25,24 @@
     {{/if}}
   {{/unless}}
 
-  <h2 class="operation-name">
+  <h2 class="operation-name {{#if (and isDeprecated (not @root.info.x-hideIsDeprecated)) }}swagger-operation-deprecated{{/if}}">
     {{md (codify summary) stripParagraph="true"}}
   </h2>
 
   {{#if description}}
     <div class="doc-row">
       <div class="doc-copy">
         <section class="swagger-operation-description operation-description">
-          {{md description}}
+          {{ md (
+              concat
+                (stripTrailing description ".")
+                (ternary (and deprecationReason (not @root.info.x-hideDeprecationReason))
+                  (concat "Deprecation Reason: " deprecationReason)
+                  ""
+                )
```

Comment:
Can you put the reason in this case above the description with its own section & class? I think just making it italic by default would be sufficient. e.g. like this:

<img width="1644" alt="screen_shot_2021-07-14_at_3 42 33_pm" src="https://user-images.githubusercontent.com/69169/125835286-d265c3dc-9805-45f9-855f-38c2fa31fc63.png">